plugin/fern_hijack.vim

```diff
@@ -8,6 +8,7 @@ function! s:hijack_directory() abort
   if !isdirectory(path)
     return
   endif
+  bwipeout %
```

Comment:
Please add `keepjumps` and `keepalt` like

```suggestion
  keepjumps keepalt bwipeout %
```

Reply:
Thank you! It's working ``vim`` and ``NeoVim``!
```
:e README.md
:e LICENSE
:e .
<C-o>

It's back to LICENSE!
```